File: src/main/resources/templates/CampaignApi/src/app/dto/registro.dto.ts
export class RegistroDto {
    username: string;
    email: string;
    password: string;
    grupo: string;

    constructor(u: string, e: string, c: string, g: string) {
        this.username = u;
        this.email = e;
        this.password = c;
        this.grupo = g;
    }
}

File: CampaingApi/src/main/java/com/salesianostriana/campaing/formbean/RegistroDto.java
package com.salesianostriana.campaing.formbean;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data @AllArgsConstructor
public class RegistroDto {
	
	private String username;
	private String email;
	private String password;
	private String grupo;	
	
}
